docs/changelog.md

```diff
@@ -4,6 +4,14 @@ title: Changelog
 sidebar_label: Changelog
 ---
 
+## Future
+
+### Major Features and Improvements
+
+### Bug Fixes
+
+* Allow for temporal types to be used with `ORDER BY` clause. [#299](https://github.com/memgraph/memgraph/pull/299)
```

Comment:
I might be too pedantic, so feel free to ignore my comment. "Allow" means more likely "adding a feature" and not fixing the bug. If I am not mistaken, memgraph just crashed in this case, so I would rephrase this:
```suggestion
* Fix crash when temporal types are used with `ORDER BY` clause. [#299](https://github.com/memgraph/memgraph/pull/299)
```